src/org/dokuwiki/translatorBundle/Services/Repository/Repository.php

```diff
@@ -467,12 +477,20 @@ private function applyChanges(GitRepository $git, $folder, TranslationUpdateEnti
         if (!$changes) throw new NoLanguageFileWrittenException();
     }
 
-    private function delete() {
-        $path = $this->buildBasePath();
+    /**
+     * Deletes the folder with the git repository checkout
+     */
+    public function deleteRepository() {
```

Comment:
Ahh that answers my previous question. Maybe `deleteCloneDirectory()` would be a better name?

Reply:
That 'repository' is a quite ambivalent name. Name is updated